### ISSUE TITLE
which version could this application run in?

### ISSUE THREAD

**Issue by lufeihaidao** [natew/obtvse#95]
Hi ,

  I meet with some problems when I tried to deploy it on my ubuntu1204 with ruby1.9.3 and rails 3.2. I guess it may be the version conflict that cause these problems?

**Comment by natew** [comment]
Haven't tested with 3.2 I don't think, but it should work otherwise.  Whats the error?

**Comment by lufeihaidao** [comment]
I haven't install the PostgreSql in my PC(Ubuntu1204 with gnome3), so I commented ''' gem 'pg', '~> 0.13' ''' in the Gemfile. And the 'bundle install' worked well as a result. But when I tried to run 'rake db:migrate' I was told that the migration was aborted.

In my virtual machine(Ubuntu1204 with Unity), I commented ''' group :production do ~~~ gem 'pg', '~> 0.13'~~~ end ''' and the 'rake db:migrate' worked well instead.

I wonder whether should I commented all the  ''' group :production do ~~~ gem 'pg', '~> 0.13'~~~ end '''  rather than  the ''' gem 'pg', '~> 0.13' ''' only?
